### DISCUSSION TITLE
Documentation of load() parameters

### DISCUSSION THREAD

**Discussion by jaclas** [sveltejs/kit#6072] [Q&A]
Where can I find a list and description of the available fields of the LayoutServerLoad object, used in the load() function of the +layout.server.js file? 
I don't see it in the documentation: https://kit.svelte.dev/docs/types#sveltejs-kit-serverload

**Comment by pilcrowonpaper** [comment]
It's in [ServerLoadEvent](https://kit.svelte.dev/docs/types#sveltejs-kit-serverloadevent)

**Comment by teemingc** [comment]
`load` fields are described here:
https://kit.svelte.dev/docs/load#input-properties

The exact types are listed here:
https://kit.svelte.dev/docs/types#sveltejs-kit-requestevent
